### PR TITLE
Hide git dir

### LINKED ISSUE #221: .git folder accessible on endofcodes.com
Please fix using .htaccess.

### PR DESCRIPTION
**Pull request by bugos**
Fixes #221.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+    - 5.5
+before_script:
+    - mysql -e 'create database endofcodes_test;'
+    - cd database/migration
+    - for i in `find .|sort|grep -v ^.$|grep -v migrate.php`; do ENVIRONMENT=test DB_NAME=travis DB_PASSWORD='' php $i; done
+    - cd ../..
+script: make all
+
+=======
+language: php
```
Comment (pkakelas) [inline .travis.yml]: Why do you repeat the script?

Comment (pkakelas) [inline .travis.yml]: You should delete lines 1 - 11. The new migrationrun proccess is completely compatible to the command in line 17.

**Comment by bugos** [comment]
Fixed. Please review again.

**Comment by dionyziz** [comment]
LGTM

**Comment by pkakelas** [comment]
@bugos good job for your first merge :)

**Comment by bugos** [comment]
@pkakelas thanks man :)

**Pull request merged**